NitroxClient/Communication/Packets/Processors/ChatMessageProcessor.cs

```diff
@@ -32,20 +32,19 @@ public override void Process(ChatMessage message)
         private void LogClientMessage(ChatMessage message)
         {
             Optional<RemotePlayer> remotePlayer = remotePlayerManager.Find(message.PlayerId);
-
-            if (remotePlayer.IsPresent())
+            if (!remotePlayer.IsPresent())
             {
-                RemotePlayer remotePlayerInstance = remotePlayer.Get();
-                ChatLogEntry chatLogEntry = new ChatLogEntry(remotePlayerInstance.PlayerName, message.Text, remotePlayerInstance.PlayerSettings.PlayerColor);
-                playerChat.WriteChatLogEntry(chatLogEntry);
-                playerChat.ShowLog();
+                return;
```

Comment:
Is this a happy case when a disconnected play sends a message and they arrive out of order?  Should we log a warn?

Reply:
Having more info when an error occurs sounds good to me.